extra/patterns/diamods.js

```diff
@@ -0,0 +1,27 @@
+function diamods(n)
+{
+    console.log('triangle for',n,'row');
+    let pattern='';
+    for(let i=1;i<=n;i++)
+    {
+        for(let j=n-1;j>=i;j--)
+        {
+        
```

Comment:
Too much unnecessary space in between code. Do proper indentation first and then submit